src/js/media-time-range.js

```diff
@@ -254,18 +256,23 @@ class MediaTimeRange extends MediaChromeRange {
     this.#boxPaddingRight = parseInt(computedStyle.getPropertyValue('--media-box-padding-right'));
 
     this.#animation = new RangeAnimation(this.range, this.#updateRange, 60);
-    this.addEventListener('transitionstart', this);
   }
 
   connectedCallback() {
     super.connectedCallback();
     this.range.setAttribute('aria-label', nouns.SEEK());
     this.#toggleRangeAnimation();
+
+    this.#rootNode = this.getRootNode();
```

Comment:
I don't want to block this PR based on adding an event listener to an ancestor, but can you put a comment here so we can keep track of this in our audit of potential GC/dangling references? Something like:
```js
// NOTE: Adding an event listener to an ancestor here.
```